cmd/controller/deploy_cluster_scope.yaml

```diff
@@ -151,8 +151,8 @@ rules:
   - connectors/status
   - attachedconnectors
   - attachedconnectors/status
-  - attachedconnectoranchors
```

Comment:
These lines need to be updated as well here:

https://github.com/skupperproject/skupper/blob/fc25f23ac8a827c301eb955ff795a82b08366f91/deployments/helm/templates/deploy_controller.yaml#L162

Reply:
Fixed